### DISCUSSION TITLE
Rotation and angles in DEG or RAD documented in cheat sheet

### DISCUSSION THREAD

**Discussion by ZlatkoKnezevic** [raysan5/raylib#2835] [Ideas]
I was toying around with simulating a car with steering and moving it around in 2D. What really tripped me up, was that e.g. DrawRectanglePro() uses angles in degrees for rotation, but Vector2Rotate() uses radians for rotation of a vector. It took me quite a while to figure out why my car was behaving very erratic.

Could it be mentioned somewhere in the cheat sheet, that a _float rotation_  is always in degrees and a _float angle_ is always in radians ( this is the pattern, that I discovered)?

**Comment by raysan5** [comment]
@ZlatkoKnezevic `raylib` library uses DEG always, easier to most newcomers. `raymath` library always uses RAD, lower-level and more general-pourpose math library, aligned with GLM library.

**Reply by ZlatkoKnezevic** [reply]
Thank you for your quick answer. Yes, I understand that degrees are easier to understand for beginners and there is nothing wrong with the way it is, but maybe it should be somewhere mentioned in the cheat sheet, so there is less confusion when you start mixing raylib and and raymath.

**Reply by chriscamacho** [reply]
I think that's a fair point - why don't you contribute to the wiki if you feel this is important
